sqldelight.bzl

```diff
@@ -10,6 +10,7 @@ def _sqldelight_codegen_impl(ctx):
         fail("Non-legacy SQLDelightc requires both module_name and package_name set.")
     args.add("--module_name", ctx.attr.module_name)
     args.add("--package_name", ctx.attr.package_name)
+    args.add("--database_name", ctx.attr.database_name)
```

Comment:
Oh oh - this shouldn't be added if there is no database name. Just put it in an if block.
```
if not ctx.attr.database_name:
    args.add(...)
```

Reply:
Also the sample app seems to be not building.

Reply:
I suspect because the default here is adding a --database_name but with nothing, so it's probably passing in an empty string - empty-string isn't null, and probably bypasses JCommander's logic around adding hte Default, and then the empty string is being naively passed around the system.

Reply:
So if you make it so you only pass --database_name when there's a non-empty one to pass, it should be fine.